packages/contracts-bedrock/src/L1/DelayedVetoable.sol

```diff
@@ -42,17 +43,11 @@ contract DelayedVetoable is ISemver {
     /// @param data The data forwarded to the target.
```

Comment:
Just a thought: this could be made into a clones-with-immutable-args factory + proxy pattern longer term, it would be cheaper to deploy minimal proxies that delegatecall a single impl, we shouldn't do this now but just something to think about

Reply:
I have a strong feeling there is room for improvement wrt the operations of this contract.